contracts/MedianOracle.sol

```diff
@@ -67,6 +73,7 @@ contract MedianOracle is OwnableUpgradeable, IOracle {
         reportExpirationTimeSec = reportExpirationTimeSec_;
         reportDelaySec = reportDelaySec_;
         minimumProviders = minimumProviders_;
+        scalar = 10**18;
```

Comment:
Should we also add a scalar param to the `init()` inputs?

Wouldn't affect our deployment, but I'd expect future deployments to consider initing it to something.

Reply:
Sounds good